setup.py

```diff
@@ -2,9 +2,13 @@
 
 from setuptools import setup
 
+with open('README.rst', 'r') as fh:
+      long_description = fh.read()
+
 setup(name='schoolopy',
-      version='0.2.0',
+      version='0.2.2',
```

Comment:
Why 0.2.2 and not 0.2.1?

Reply:
Because the project description will update only if you create a new release.